x-pack/plugins/security_solution/public/common/components/item_details_card/index.tsx

```diff
@@ -66,16 +66,13 @@ const DescriptionListDescription = styled(EuiDescriptionListDescription)`
 interface ItemDetailsPropertySummaryProps {
   name: ReactNode | ReactNode[];
   value: ReactNode | ReactNode[];
-  title?: string;
 }
 
 export const ItemDetailsPropertySummary: FC<ItemDetailsPropertySummaryProps> = memo(
```

Comment:
I belive that when you define the Type using `FC<>` generic, it automatically supports `children` prop, which is not the case here. Can you change to this instead:

```suggestion
export const ItemDetailsPropertySummary = memo<ItemDetailsPropertySummaryProps>(
```

There may be other components setup this way where this should also be done. Feel free to open up a separate Issue to track :)